pentaho-karaf-assembly/src/main/filtered-resources/etc/custom.properties

```diff
@@ -140,6 +140,13 @@ org.apache.commons.logging.*; version\="1.1.3", \
  io.reactivex.*;version\="2.0.4", \
  org.pentaho.di.engine.*;version="${pdi.version}", \
  org.reactivestreams.*;version="1.0.0", \
- org.apache.hadoop.hbase.client.*
+ org.apache.hadoop.hbase.*, \
+ org.apache.hadoop.hbase.client.*, \
+ org.apache.hadoop.hbase.io.*, \
+ org.apache.hadoop.hbase.mapreduce.*, \
+ org.apache.hadoop.hbase.protobuf.*, \
+ org.apache.hadoop.hbase.protobuf.generated.*, \
+ org.apache.hadoop.hbase.util.*,\
+ com.google.common.collect;version\="12.0.0"
```

Comment:
I made some comments on the bb-8 channel as I would like to understand the reason to keep importing these packages.